src/content/reference/similar-ec2-instance-types/index.md

```diff
@@ -14,6 +14,8 @@ owner:
 
 # Similar EC2 Instance Types
 
+{{< platform_support_table aws="ga=11.2.0" >}}
+
```

Comment:
This is unrelated to the main purpose of the PR. It will add this box:

![image](https://user-images.githubusercontent.com/273727/124938161-5bd55580-e008-11eb-94c1-3f98c08e5dee.png)